.travis.yml

```diff
@@ -14,6 +15,7 @@ env:
   - PHPCS_BRANCH=2.2.0
 
 matrix:
+  fast_finish: true
```

Comment:
It doesn't cause the build to stop at the first failure, it just causes the notification to be sent as soon as there is a failure instead of waiting for the build to finish. (In case that was what you meant by it not having any effect.)